milestone1.css

```diff
@@ -0,0 +1,55 @@
+.recipes {
+    font-family: "Papyrus";
+    background-color: rgb(219, 199, 245);
+    padding: 10px;
+    margin: 10px;
+    border: 1px rgb(219, 199, 245);
+    border-radius: 10px;
+}
+
+.header {
+    font-family: "Papyrus";
+    background-color: rgb(120, 75, 152);
+    padding: 50px
+    margin: 10px
+    border: rgb(120, 75, 152);
+}
+
+.title-section {
+    font-family: "Papyrus";
+    background-color: rgb(120, 75, 152);
+    padding: 60px
+    margin: 10px
+    border: 10px rgb(120, 75, 152);
+}
+
+.navigation {
+    display: flex;
+    justify-content: flex-end;
+    font-family: "Papyrus";
+    background-color: rgb(120, 75, 152);
+    padding: 60px
+    margin: 10px
+    border: 5px solid black;
+
+.recipe-images {
+    margin: 10px
+    padding: 50px
+    border-radius: 10px
+    font-family: "Papyrus";
+}
+
+.recipe-content {
+    font-family: "Papyrus";
+    display: flex;
+    justify-content: flex-end;
```

Comment:
Not sure what you're going for with the flex-end here, did you experiment with the other flexbox options?